compiler/hash-typecheck/src/storage/primitives.rs

```diff
@@ -110,14 +118,19 @@ impl Scope {
 
     /// Add a member to the scope, overwriting any existing member with the same
     /// name.
-    pub fn add(&mut self, member: Member) {
+    pub fn add(&mut self, member: Member) -> usize {
```

Comment:
Can return the scope member here.

Reply:
There is no access `scope_id` which `ScopeMember` requires